src/main/windows/editor.js

```diff
@@ -96,6 +96,12 @@ class EditorWindow extends BaseWindow {
       win.webContents.send('AGANI::window-active-status', { status: false })
     })
 
+    ;['maximize', 'unmaximize', 'enter-full-screen', 'leave-full-screen'].forEach(channel => {
+      win.on(channel, () => {
+        win.webContents.send(`window-${channel}`)
```

Comment:
I think we need to prefix the `channel` name with `mt::`

Reply:
Fixed, I also updated readme and my upcoming PR also improve startup performance a bit. Removing `win.on(...)` results in a way faster initial vue mount time (avg. 0.45s before and 0.028s after). I think due a `source-map-support` issue that parsed the `electron` code.